.github/workflows/build-test-package-python-cuda.yml

```diff
@@ -90,6 +90,35 @@ jobs:
         echo "Searching for wheels matching pattern ${WHEEL_PATTERN}"
         python -m twine check ${WHEEL_PATTERN}
 
+    - name: Install test dependencies
+      shell: bash
+      run: |
+        if [[ -f ./test/requirements.txt ]]; then
+          python -m pip install -r ./test/requirements.txt
+        fi
+        python -m pip install pytest nbmake
+
+    - name: Download Python Package Artifacts
+      uses: actions/download-artifact@v4
+      with:
+        name: LinuxWheel3${{ matrix.python3-minor-version }}_2_28-x64
```

Comment:
@acoussat The correct name is at line 109... I don't know if you can use wild cards here? To be checked...